### PR TITLE
[vcpkg] Use spaces instead of semicolons in the output

### PR DESCRIPTION
**Pull request by Pospelove**
If the package build fails, we get an error message like this:
```
-- Building x86-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/vcpkg/downloads/tools/cmake-3.14.0-windows/cmake-3.14.0-win32-x86/bin/cmake.exe;--build;.;--config;Debug;--;/p:VCPkgLocalAppDataDisabled=true;/p:UseIntelMKL=No;/m
```
For readability, it would be better to use spaces instead of semicolons to separate command arguments.
```
-- Building x86-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:135 (message):
    Command failed: C:/vcpkg/downloads/tools/cmake-3.14.0-windows/cmake-3.14.0-win32-x86/bin/cmake.exe --build . --config Debug -- /p:VCPkgLocalAppDataDisabled=true /p:UseIntelMKL=No /m
```

### CODE REVIEW

**Comment by cskrisz** [comment]
* Command arguments containing spaces should be wrapped in quotes, so we know that they are in fact, a single argument
* `vcpkg_execute_required_process` and `vcpkg_execute_required_process_repeat` should also use this format then

**Comment by Pospelove** [comment]
@cskrisz Thanks for your feedback. Please review again.

File: scripts/ports.cmake [hunk 1]
```diff
@@ -6,6 +6,18 @@ macro(debug_message)
     endif()
 endmacro()
 
+macro(prettify_command INPUT_VAR OUTPUT_VAR)
```
Comment (ras0219-msft) [inline scripts/ports.cmake]: This should live in a `scripts/cmake/vcpkg_prettify_command.cmake` file and be renamed to `vcpkg_prettify_command` to meet our helper conventions. Adding a short help blob[1] to the top similar to other commands would also be great!

[1] https://github.com/microsoft/vcpkg/blob/master/scripts/cmake/vcpkg_add_to_path.cmake

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: scripts/ports.cmake (hunk 1 above)
Comment (Pospelove) [reply scripts/ports.cmake]: Fixed in 14d805d497d36178413fedafe9fb9b07aa62b799.

**Comment by vicroms** [comment]
Thanks for the PR!

**Comment by Voskrese** [comment]
@vicroms 
meson error

vcpkg_install_meson.cmake

`COMMAND ${NINJA} install -v`

failed
    `vcpkg_execute_required_process(`

ok
    `execute_process(`

**Comment by Voskrese** [comment]
false alarm
can't figure out why python gives an error
```

"cp" "-f" "../src/tk-doc-zip-b6950d3b86/gtkdoc/__init__.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/check.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/common.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/fixxref.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/highlight.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/md_to_db.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/mkdb.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/mkhtml.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/mkhtml2.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/mkman.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/mkpdf.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/rebase.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/scan.py" "../src/tk-doc-zip-b6950d3b86/gtkdoc/scangobj.py" "gtkdoc"
CreateProcess failed: The system cannot find the file specified.
ninja: build stopped: subcommand failed.
```
Apparently in this, the CP team
"cp" "-f"

**Pull request merged**